app/src/components/v-text-overflow.vue

```diff
@@ -11,14 +11,15 @@ import { useElementSize } from '@directus/composables';
 
 interface Props {
 	/** The text that should be displayed */
-	text: string | number | boolean | Record<string, any> | Array<any>;
+	text?: string | number | boolean | Record<string, any> | Array<any>;
 	/** What parts of the text should be highlighted */
 	highlight?: string;
 	/** The placement of the tooltip */
 	placement?: 'top' | 'bottom' | 'left' | 'right' | 'start' | 'end';
 }
 
 withDefaults(defineProps<Props>(), {
+	text: '',
```

Comment:
The warning you're currently seeing in `/admin/settings/data-model/whatever` actually seems to be more of an indication that the components are being rendered even though the data is not yet loaded... In principle, however, it's okay for me to accept an undefined value here. Should we default to `undefined` instead of `''`though?